main/sw/source/filter/html/swhtml.cxx

```diff
@@ -5136,18 +5119,18 @@ void SwHTMLParser::InsertLineBreak()
 
 	if( !bCleared && !bBreakItem )
 	{
-		// wenn kein CLEAR ausgefuehrt werden sollte oder konnte, wird
-		// ein Zeilenumbruch eingef?gt
+		// if no CLEAR should be executed or could be executed
+		// a line break is inserted
 		String sTmp( (sal_Unicode)0x0a );	// make the Mac happy :-)
         pDoc->InsertString( *pPam, sTmp );
 	}
 	else if( pPam->GetPoint()->nContent.GetIndex() )
 	{
-		// wenn ein Claer in einem nicht-leeren Absatz ausgefuehrt wurde,
-		// muss anschliessen ein neuer Absatz aufgemacht werden
-		// MIB 21.02.97: Eigentlich muesste man hier den unteren Absatz-
-		// Absatnd auf 0 drehen. Das geht aber bei sowas wie <BR ..><P>
-		// schief (>Netacpe). Deshalb lassen wir das erstmal.
+		// if a claer was executed in a non-empty paragraph,
+		// a new paragraph must be opened afterwards.
+		// MIB 21.02.97: Actually, the lower paragraph should be set to 0.
+		// But this works with something like <BR ..><P>.
+		// (>Netacpe). Therefore we leave it for now.
 		AppendTxtNode( AM_NOSPACE );
```

Comment:
If a **Clear** was executed in a non-empty paragraph, **then** a new paragraph must be opened afterwards.
MIB 21.02.97: actually, the lower paragraph **spacing** should be set to 0. But this works with something like `<BR ..><P>` (>**Netscape**). Therefore we leave it for now.